README.md

```diff
@@ -48,12 +55,12 @@ Here's how to set up and run the Pac-Man Contribution Graph:
     import { generatePacManGame } from 'pacman-contribution-graph';
 
     generatePacManGame({
-    	username: 'your_username',
```

Comment:
Can you fix all linting issues. Spacing is different from original code. You can run `pnpm exec prettier --check "**/*.{ts,json,md}"` to check all current linting issues